code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -223,49 +223,51 @@
+	desc = "An okayish single-malt whiskey. This one is produced mainly in New Valletta on Callisto and is fairly famous among Cythereans, too. It's great to get you \
+	in the right mindset for your tenth night out clubbing in a row!"
 	desc_fluff = DRINK_FLUFF_GETMORE
 	icon_state = "whiskeybottle"
 	center_of_mass = list("x"=16, "y"=4)
 	reagents_to_add = list(/decl/reagent/alcohol/whiskey = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/fireball
 	name = "Delta Cephei Cinnamon Fireball"
-	desc = "An okayish single-malt whiskey, infused with cinnamon and hot pepper inside the maintenance tunnels of NSS Cepheus. \
-			It is sometimes claimed that Tunnel Runners came up with the current recipe for this drink."
+	desc = "An okayish single-malt whiskey, infused with cinnamon and hot pepper that used to be mainly produced on Mars, but the production line was since moved to Earth for geopolitical reasons. \
+			It is sometimes claimed that particularly desperate Eridanian dregs came up with the current recipe for this drink."
 	desc_fluff = DRINK_FLUFF_GETMORE
 	icon_state = "whiskeybottle"
 	center_of_mass = list("x"=16, "y"=4)
 	reagents_to_add = list(/decl/reagent/alcohol/fireball = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/vodka
 	name = "Martian 50% Premium"
-	desc = "Only potatoes grown in real imported Martian soil may be used for this premium vodka. Made by Silverport, drunk by Zavodskoi."
+	desc = "Only potatoes grown in real imported Martian soil may be used for this premium vodka (imports of Martian soil may have stopped). Made by Silverport, drunk by Zavodskoi."
 	desc_fluff = DRINK_FLUFF_SILVERPORT
 	icon_state = "vodkabottle"
 	center_of_mass = list("x"=17, "y"=4)
 	reagents_to_add = list(/decl/reagent/alcohol/vodka = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/vodka/mushroom
-	name = "Magnitogorsk Import mushroom vodka"
-	desc = "A mushroom-based vodka imported from the breweries of Himeo. Drinking too much of this will result in a personal permanent revolution."
+	name = "Inverkeithing Import mushroom vodka"
+	desc = "A mushroom-based vodka imported from the breweries of Inverkeithing on Himeo. Drinking too much of this will result in a personal permanent revolution."
 	desc_fluff = "Vodka made from mushrooms is a local favourite on Himeo, due to the ease with which mushrooms can be grown under the planet's surface. This bottle is \
-	from the Vodnik Breweries Syndicate in Magnitogorsk, arguably the most famous brewery on Himeo (though fans of the Coal Canary Syndicated Brewery in Highland City \
-	would disagree). Drinkers of the world, unite!"
+	from the Schwarzer Drache Breweries Syndicate in Inverkeithing, arguably the most famous brewery on Himeo due to its historical reputation. It is also the most famous brand \
+	of mushroom vodka among non-Himeans because of Inverkeithing's developing tourism industry. Drinkers of the world (and beyond), unite!"
```

Comment:
```suggestion
	of mushroom vodka among non-Himeans because of Inverkeithing's developing tourism industry. Drinkers of the world and beyond, unite!"
```